jsonschema_gentypes/validate.py

```diff
@@ -111,21 +111,29 @@ def format_error(error: jsonschema.exceptions.ValidationError) -> str:
             parent = None
             if hasattr(curent_data, "lc"):
                 parent = curent_data
-            for path in error.path:
+            for path in error.absolute_path:
                 curent_data = curent_data[path]
                 if hasattr(curent_data, "lc"):
                     parent = curent_data
             if parent is not None:
                 position = f"{filename}:{parent.lc.line + 1}:{parent.lc.col + 1}"  # type: ignore
 
-        return (
-            f" - {position} "
-            f'({".".join([str(i) for i in error.path] if error.path else "/")}): '
-            f"{error.message} (rule: "
-            f'{".".join([str(i) for i in error.schema_path] if error.schema_path else "/")})'
-        )
-
-    return sorted([format_error(e) for e in validator.iter_errors(data)]), data
+        if error.context:
+            result = []
```

Comment:
small things, but `results` would be better (with a `s` at the end)
Same at line 133

Reply:
+1 Thanks